File: fixtures/elixir_output/post_json_multiple_headers.ex
request = %HTTPoison.Request{
  method: :post,
  url: "https://0.0.0.0/rest/login-sessions",
  options: [hackney: [:insecure]],
  headers: [
    {~s|Content-Type|, ~s|application/json|},
    {~s|X-API-Version|, ~s|200|},
  ],
  params: [],
  body: ~s|{"userName":"username123","password":"password123", "authLoginDomain":"local"}|
}

response = HTTPoison.request(request)

File: test/fixtures/node-got/post_json_multiple_headers.js
import got from 'got';

const response = await got.post('http://localhost:28139/rest/login-sessions', {
  headers: {
    'Content-Type': 'application/json',
    'X-API-Version': '200'
  },
  // body: '{"userName":"username123","password":"password123", "authLoginDomain":"local"}',
  json: {
    'userName': 'username123',
    'password': 'password123',
    'authLoginDomain': 'local'
  },
  https: {
    rejectUnauthorized: false
  }
});
